geometry_msgs/Quaternion orientation
geometry_msgs/Vector3 body_rates
geometry_msgs/Vector3 linear_acceleration